.github/workflows/run-tests.yml

```diff
@@ -37,10 +37,10 @@ jobs:
     - name: Docker Compose
       run: docker-compose up --build -d
 
-    - name: Sleep for 30 seconds
+    - name: Sleep for 90 seconds
       uses: juliangruber/sleep-action@v1
       with:
-        time: 30s
+        time: 90s
```

Comment:
Whats this used for?

Reply:
Changed because 2 extra projects are started. Earlier when the sleep was still 30s the Postman test seemed to start when applications were not started yet.
After changing to 90s it worked immediately, so that's why we changed it.